plonetheme/onegovbear/profiles/default/cssregistry.xml

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0"?>
 <object name="portal_css">
 
+    <!-- Disable member.css, because the "expression" and the "authenticated"
+     flag cannot be combined.
+     It results in ignoring the expression, which means we still have an
+     additional resource for member.css, which we dont want.
+     /-->
+    <stylesheet id="member.css" enabled="False" />
```

Comment:
Do we need an upgrade step for this one?
I'm not sure because it was already a problem before and we have fixed it in policies..
@maethu inputs?